lib/guardian.rb

```diff
@@ -307,6 +307,14 @@ def can_grant_title?(user, title = nil)
     user.groups.where(title: title).exists?
   end
 
+  def can_have_primary_group?(user, group_id=nil)
```

Comment:
I think we should change this method name to something more clear. I suggest `can_use_primary_group` or `can_show_primary_group`.

Reply:
Use is great. this naming tormented me.